packages/@aws-cdk/aws-apigateway/lib/stage.ts

```diff
@@ -166,25 +183,49 @@ export class Stage extends Resource {
   public readonly restApi: IRestApi;
   private enableCacheCluster?: boolean;
 
+  private readonly accessLogDestination?: IAccessLogDestination;
+  private readonly accessLogFormat?: IAccessLogFormat;
+
   constructor(scope: Construct, id: string, props: StageProps) {
     super(scope, id);
 
     this.enableCacheCluster = props.cacheClusterEnabled;
 
     const methodSettings = this.renderMethodSettings(props); // this can mutate `this.cacheClusterEnabled`
 
+    // custom access logging
+    let accessLogSetting: CfnStage.AccessLogSettingProperty | undefined;
+    this.accessLogDestination = props.accessLogDestination;
+    this.accessLogFormat = props.accessLogFormat;
+    if (this.accessLogDestination == null && this.accessLogFormat == null) {
```

Comment:
We don't use `null`. We normally use `undefined`.

```suggestion
    if (!this.accessLogDestination && !this.accessLogFormat) {
```

Change in other place below as well.